### PR TITLE
Review and Updates to Communication

### PR DESCRIPTION
**Pull request by RicardoCrespo-ora**
Description
----
In preparation for migrating to Oracle docs, we are reviewing and correcting any inconsistencies, typos, broken links, etc. This PR aims to correct any of the above that needs to be addressed within Communication and any other information directly referenced by it.

### Linking to Encounter and Patient
**Before**
image

**After**
image

### Capitalizacion fixes

**Before**
image

image

image

image

image

image

**After**

image

image

image

image

image

### Parameters table formatting

**Before**
image

image

**After**

image

*on scroll*
image

image

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [x] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by cldeitrich — APPROVED** [review]
[PRODOC-203712](https://jira.cerner.com/browse/PRODOC-203712): Approved for style.

File: content/millennium/r4/clinical/request-and-response/communication.md [hunk 1]
```diff
@@ -39,41 +45,41 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
 
 ID             | Value\[x] Type      | Description
 ---------------|---------------------|----------------------------------------------------------------------------------
-`reply-to`     | [`Reference`]       | Used to direct where replies to the communication should be sent. Must be a Group.
-`email-status` | [`CodeableConcept`] | Status of the electronic communication (in-progress, completed, received).
+`reply-to`     | [`Reference`]       | Directs where replies to the communication are sent. Must be a group.
+`email-status` | [`CodeableConcept`] | The status of the electronic communication (in-progress, completed, or received).
```
Comment (amcginn) [inline content/millennium/r4/clinical/request-and-response/communication.md:49]: I forget, have we been listing the required values here or in the notes section?

File: content/millennium/r4/clinical/request-and-response/communication.md [hunk 2]
```diff
@@ -39,41 +45,41 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
...
+Search for communications that meet supplied query parameters.
 
     GET /Communication?:parameters
 
-_Implementation Notes_
+_Notes_
 
-* The Content of a communication will be returned through the binary resource. A reference to the payload will be provided in the Payload section.
-* Only 1000 elements max will be returned based on the date range.
+* The content of a communication is returned through the Binary resource. A reference to the payload is provided in the Payload section.
```
Comment (amcginn) [inline content/millennium/r4/clinical/request-and-response/communication.md:59]:
```suggestion
* The content of a communication is returned through the Binary resource. A reference to the payload is provided in `Communication.payload.contentReference`.
```

File: content/millennium/r4/clinical/request-and-response/communication.md [hunk 3]
```diff
@@ -175,21 +174,15 @@ X-Runtime: 0.022706
...
 
 Patch an existing communication.
 
     PATCH /Communication/:id
 
-_Implementation Notes_
+_Notes_
 
-* This implementation follows the [JSON PATCH](https://tools.ietf.org/html/rfc6902) spec.
+* This implementation follows the [JSON PATCH](https://tools.ietf.org/html/rfc6902) specification.
```
Comment (amcginn) [inline content/millennium/r4/clinical/request-and-response/communication.md:185]:
```suggestion
* This implementation follows RFC 6902, [JSON Patch](https://tools.ietf.org/html/rfc6902).
```

**Review by amcginn — APPROVED** [review]

**Comment by SukeshKonjeti** [comment]
after changes published
Screenshot 2024-02-02 at 5 25 13 PM

Screenshot 2024-02-02 at 5 25 40 PM

**Pull request merged**